doc/example-config-nginx.md

```diff
@@ -0,0 +1,74 @@
+# How to run Spreed WebRTC with Nginx in subpath
+
+To support Spreed WebRTC with an Nginx HTTP Server, you require at least Nginx
+version 1.3.13 to be able to proxy Websocket protocol. All current distributions
+should have this in their respositories.
+
+Add the configuration section from below to your Nginx configuration, to make
+available Spreed WebRTC below the `/webrtc` subpath. Note that Spreed WebRTC does
+require HTTPS and thus it only make sense to use and TLS encrypted `server`.
+
+## Nginx configuration sniplets
+
+### http context
+
+The following section, needs to go inside the `http` context of your Nginx
+configuration. This enables Websocket proxy - make sure you have it only once.
+
+```
+	map $http_upgrade $connection_upgrade {
```

Comment:
You can even use
\`\`\`nginx
[..]
\`\`\` to use nginx config syntax highlighting.